modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -77,20 +77,19 @@ Node *EditorSceneFormatImporterBlend::import_scene(const String &p_path, uint32_
 	} else {
 		parameters_arg += "export_extras=False,";
 	}
-	if (p_options.has(SNAME("blender/meshes/skins")) && p_options[SNAME("blender/meshes/skins")]) {
+	if (p_options.has(SNAME("blender/meshes/skins"))) {
 		int32_t skins = p_options["blender/meshes/skins"];
 		if (skins == BLEND_BONE_INFLUENCES_NONE) {
-			parameters_arg += "export_all_influences=False,";
+			parameters_arg += "export_skins=False,";
 		} else if (skins == BLEND_BONE_INFLUENCES_COMPATIBLE) {
-			parameters_arg += "export_all_influences=False,";
+			parameters_arg += "export_all_influences=False,export_skins=True,";
 		} else if (skins == BLEND_BONE_INFLUENCES_ALL) {
-			parameters_arg += "export_all_influences=True,";
+			parameters_arg += "export_all_influences=True,export_skins=True,";
 		}
-		parameters_arg += "export_skins=True,";
```

Comment:
I'm not familiar with mesh skinning in Blender but it's not 100% clear to me how the enum values match the used options. I assume that if `export_skins` is `False` then `export_all_influences` isn't used at all?

Reply:
@julienduroure I think that's correct, but maybe you have something to add.

Reply:
That is correct, it's an option related to skinning, so if skinning is disabled it has no effect.
In the exporter it's grayed out if skinning is off:
![Screenshot_20221004_123232](https://user-images.githubusercontent.com/44414418/193875174-22bb02ec-feb2-4671-bbb0-210a55d30ee8.png)
Also this is the same arguments as the `else` statement that it was going to before. I made sure to test all the options, they work as expected.